package.json

```diff
@@ -22,7 +22,11 @@
     "@angular/core": "~4.0.3",
     "@angular/platform-browser": "~4.0.3",
     "@angular/platform-browser-dynamic": "~4.0.3",
+    "babel-preset-react": "^6.24.1",
     "core-js": "~2.4.1",
+    "prop-types": "^15.5.10",
+    "react": "^15.6.1",
```

Comment:
please update react/react-dom to 15.6.2 and prop-types to 15.6.0 as they now uses MIT license.